src/cli.ts

```diff
@@ -138,6 +139,12 @@ function makeCommand (process: NodeJS.Process): Command {
     ).default(
       Enums.ComponentType.Application
     )
+  ).addOption(
+    new Option(
+      '--include-url-qualifiers',
+      'Include URL qualifiers in PURL.\n' +
+      'URL qualifiers such as download_url and vcs_url can cause long PURLs and are omitted by default'
```

Comment:
put the qualifier's names in double quotes (`"`)

Reply:
👍